mainchain-follower/db-sync-follower/src/db_model.rs

```diff
@@ -527,3 +527,36 @@ pub(crate) async fn index_exists(pool: &Pool<Postgres>, index_name: &str) -> boo
 		.map(|rows| rows.len() == 1)
 		.unwrap()
 }
+
+#[derive(Debug, Clone, sqlx::FromRow, PartialEq)]
+pub(crate) struct ReleasedTokens(pub Option<StakeDelegation>);
```

Comment:
I used `StakeDelegation` here because it represents a number of tokens, but obviously we'll need either some new type or rename `StakeDelegation` to something universal like `Lovelace`

Reply:
Why would you use `Lovelace` or same type as `StakeDelegation` for native tokens? They are different types even by allowed range as far as I know.

Reply:
because I'm a dumbass and primed myself to think it makes sense

Reply:
fixed that silliness